### ISSUE TITLE
Unclear dependency: 'vision'

### ISSUE THREAD

**Issue by neuneck** [davidbunk/EasyLabelKit#1]
Hi! 
I am always looking for the best tool to label images and came across your EasyLabelKit. I'd love to give it a shot, but when launching elk.py with python 3.6.6 on Ubuntu 18.04 LTS I run into a `ModuleNotFoundError`:

```
$ python3 elk.py
>>> Traceback (most recent call last):
>>>   File "elk.py", line 13, in <module>
>>>     from gui import Ui_MainWindow
>>>   File "/home/julian/git_repos/EasyLabelKit/gui.py", line 243, in <module>
>>>     from vision import MplCanvas, Treeview
>>> ModuleNotFoundError: No module named 'vision'
```

A simple search for 'vision' on pypi did not reveal an obvious candidate. Clarification on this dependency (and possible a requirements.txt) would be much appreciated!

**Comment by ravenbunk** [comment]
Hi @neuneck,

thanks for pointing this out. This is an internal import that I just forgot to rename. The issue should be fixed with 55e1babb5d2a01e78b8f99371e974cc87e9cd773 .

Best, David

**Comment by neuneck** [comment]
I can confirm the issue is fixed. Thanks!
